dvc/config.py

```diff
@@ -2,16 +2,20 @@
 
 from __future__ import unicode_literals
 
-from dvc.utils.compat import str, open
-
 import os
 import re
 import copy
 import errno
+import logging
 import configobj
+from schema import Schema, Optional, And, Use, Regex, SchemaError
+
+
+from dvc.utils.compat import str, open
+
+
 import logging
```

Comment:
Since you've moved `import logging` up already, there is no need for the second one.
```suggestion
```